reference/mouse.md

```diff
@@ -226,3 +226,95 @@ The `sampling_period` parameter is expressed in milliseconds, and defines how fr
 Note that the first state will be available only after the first sampling period elapsed.
 After that, the state can be read by calling the `wb_mouse_get_state` function (for more details, see [`WbMouseState`](#wbmousestate)).
 The `wb_mouse_disable` function should be used to stop the mouse readings.
+
+
+#### `wb_mouse_enable_3d_position`
+#### `wb_mouse_disable_3d_position`
+#### `wb_mouse_is_3d_position_enabled`
+
+%tab-component
+
+%tab "C"
+
+```c
+#include <webots/mouse.h>
+
+void wb_mouse_enable_3d_position();
+void wb_mouse_disable_3d_position();
+bool wb_mouse_is_3d_position_enabled();
+```
+
+%tab-end
+
+%tab "C++"
+
+```cpp
+#include <webots/Mouse.hpp>
+
+namespace webots {
+  class Mouse {
+    void enable3dPosition();
+    void disable3dPosition();
+    bool is3dPositionEnabled() const;
+  }
```

Comment:
When not all methods are included, we use to to add a line like:
```c++
// ...
```
This is use many times in the reference manual, we should keep consistent with this (see for example the Accelerometer API).